lib/snapshot.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+const path = require('path');
+const fs = require('fs');
+const isEqual = require('lodash.isequal');
+const mkdirp = require('mkdirp');
+const globals = require('./globals');
+
+class Snapshot {
+	constructor(testPath, options) {
+		if (!testPath) {
+			throw new TypeError('Test file path is required');
+		}
+
+		this.dirPath = path.join(path.dirname(testPath), '__snapshots__');
```

Comment:
> I would really prefer not introducing more options. Can we make it __snapshots__ if inside a __tests__ directory, otherwise snapshots?

Yea that sounds great to me.